### ISSUE TITLE
Error occurred when passing backtick in component prop

### ISSUE THREAD

**Issue by stacysun-cb** [figma/code-connect#139]
Code Connect CLI version: 1.0.6
Operating system: macOS

Hello team! I try to pass custom css inside backticks to a classname prop `dangerouslySetContentClassName`, but it results in error on Figma. Here's the code connect file: 

```
figma.connect(
  FullscreenModal,
  'https://www.figma.com/...',
  {
    variant: { layout: 'primary centered' },
    props: {
      secondaryContent: figma.instance('🔄 secondary content'),
      primaryContent: figma.instance('🔄 primary content'),
    },
    example: (props) => {
      const [visible, { toggleOn, toggleOff }] = useToggler(true);
      const { triggerRef, focusTrigger } = useTriggerFocus();

      return (
        <>
          <Button ref={triggerRef} onPress={toggleOn}>
            Open Modal
          </Button>
          <FullscreenModal
            closeAccessibilityLabel="Close modal"
            dangerouslySetContentClassName={css`
              margin-left: auto;
              justify-content: center;
            `}
            onDidClose={focusTrigger}
            onRequestClose={toggleOff}
            title="Modal title"
            visible={visible}
            {...props}
          />
        </>
      );
    },
  },
);
```
My questions:
- How to render prop value with backticks?
- This code connect file renders both the visual and functionality for FullscreenModal component. On Figma design, there's is no Button component, but just a FullScreenModal. I wonder what's the best practice to code connect components like this? Shall we consider functionality at all or just render the code that reflects the design?

Thank you!

**Comment by karlpetersson** [comment]
Hey @stacysun-cb! Thanks for reporting this, the backtick issue is a bug and will be fixed in the coming release. Using backticks in the `example` function will break the code snippet currently. Unfortunately I don't have a workaround to suggest other than omitting the prop for now.

As for your other question, it really is up to you, but I'd personally suggest that the code snippet should consider and include all functionality that's relevant in using the component. We often see people pass example state and handlers to illustrate usage, which I think is good practice.

**Comment by tomduncalf-figma** [comment]
Hey @stacysun-cb, we just released Code Connect v1.1.1 which should resolve this issue. Please feel free to reopen the issue or create a new one if you continue to have issues.
